src/types/siteLaunch.ts

```diff
@@ -40,6 +40,19 @@ export interface SiteLaunchMessage {
   statusMetadata?: string
 }
 
+export type SiteLaunchResult = Omit<
+  SiteLaunchMessage,
+  | "status"
+  | "statusMetadata"
+  | "redirectionDomain"
+  | "appId"
+  | "requestorEmail"
+  | "agencyEmail"
+> & {
+  redirectionDomainSource?: string
+  redirectionDomainTarget?: string
```

Comment:
is this just to facilitate the initial construction of `successResult` and setting the props later? it seems like we'll always have the properties (we throw on error and we have a default) so maybe we should set this as required to better reflect reality

Reply:
no ah no all site launches use redirection server leh